cmd/juju/application/deployer/charm.go

```diff
@@ -550,11 +550,13 @@ func isEmptyOrigin(origin commoncharm.Origin, source commoncharm.OriginSource) b
 func formatLocatedText(curl *charm.URL, origin commoncharm.Origin) string {
 	repository := origin.Source
 	if repository == "" || repository == commoncharm.OriginLocal {
-		return fmt.Sprintf("Located local charm %q, revision %d", curl.Name, curl.Revision)
+		return fmt.Sprintf("Located local charm %q, next %d", curl.Name, curl.Revision)
```

Comment:
typo

Reply:
thanks, got caught by IDE refactor, caught one of them, just not the 2nd.